app/controllers/journals_controller.rb

```diff
@@ -0,0 +1,2 @@
+class JournalsController < ApplicationController
+end
```

Comment:
we need the journal model as well as we have the journal controller and journal table